ARCs/arc-0013.md

```diff
@@ -0,0 +1,96 @@
+
+## Abstract
+
+The algorand blockchain is decentralized in the form that any person can create wallet, and use its assets for his or her own good. There are many methods how to store data to the blockchain, for example the ARC-0002 standard using the transation note field, or by creating the referenced document, or by providing data through the smart contracts. 
+
+Developers can create their own standards how they store or exchange data between applications.
+
+This ARC aims to bring together the developers to the central point of knowledge interchange and guides them how to create standards.
+
+## Specification
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+ARC-0002 is the [Algorand Transaction Note Field Conventions](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0002.md) standard
+
+JSON is the standard described in [RFC-7159](https://www.ietf.org/rfc/rfc7159.txt).
+
+### Standard live
+
+Idea - An idea that is pre-draft. This is not tracked within the Repository.
+
+Draft - The first formally tracked stage in development. Standard is merged by an Algorand foundation git administrator into the repository when properly formatted.
+
+Review - A standard's Author marks a standard as ready for and requesting Peer Review.
+
+Last Call - This is the final review window for a standard before moving to FINAL. A foundation editor will assign Last Call status and set a review end date (review-period-end), typically 14 days later.
+
+If this period results in necessary normative changes it will revert a standard to REVIEW.
+
+Final - This standard represents the final standard. A Final standard exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+
+Stagnant - Any standard in DRAFT or REVIEW if inactive for a period of 6 months or greater is moved to STAGNANT. A standard may be resurrected from this state by Authors or foundation Editors through moving it back to DRAFT.
+
+Standard Authors are notified of any algorithmic change to the status of their standard
+
+Withdrawn - A standard Author(s) have withdrawn the proposed standard. This state has finality and can no longer be resurrected using this standard number. If the idea is pursued at later date it is considered a new proposal.
+
+Living - A special status for standard that are designed to be continually updated and not reach a state of finality.
+
+### Algorand foundation role
+
+Algorand foundation manages list of standards provided by the community and assigns them a standard identifier.
+
+### List of the Community Algorand Standards
+
+List of standards is available here: https://github.com/algorandfoundation/ARCs/blob/main/ARCs/CAS.json
```

Comment:
this link 404s

Reply:
it is 404 because the file is in this pull request